client/state/plugins/wporg/actions.js

```diff
@@ -12,35 +13,38 @@ import wporg from 'lib/wporg';
 import { normalizePluginData } from 'lib/plugins/utils';
 import { WPORG_PLUGIN_DATA_RECEIVE, FETCH_WPORG_PLUGIN_DATA } from 'state/action-types';
 
-/**
- *  Local variables;
- */
-const _fetching = {};
+// TODO: fix the selector in `selectors.js` to not return `true` by default
+function isFetching( state, pluginSlug ) {
```

Comment:
Temporary addition that will be removed when the Plugins store and UI gets Reduxified. I expect that the version of the `isFetching` selector that returns `true` by false will disappear. The UI will show placeholder when the plugin info in state is `null`.

Then, if we rewrite the fetching action to data layer, maintaining `isFetching` won't be needed any more. Because the fetching action uses an unusual fetching mechanism (JSONP), the rewrite might be harder than usual.